src/style.css

```
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  --active-color: #00acdc;
+}
+
```

variables should be in `:root`
```suggestion
:root {
  --active-color: #00acdc;
}

body {
  margin: 0;
}

```